server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionTests.java

```diff
@@ -506,7 +512,8 @@ public void testMasterFailoverAfterStepDown() throws ExecutionException, Interru
         new Action("internal:testAction", transportService, clusterService, threadPool) {
             @Override
             protected void masterOperation(Request request, ClusterState state, ActionListener<Response> listener) throws Exception {
-                // The other node has become master, simulate failures of this node while publishing cluster state through ZenDiscovery
+                // The other node has become cluster-manager, simulate failures of this node while publishing cluster state through
+                // ZenDiscovery
```

Comment:
`testDelegateToMasterOnNodeWithDeprecatedMasterRole` -> `testDelegateToClusterManagerOnNodeWithDeprecatedClusterManagerRole` and the comment above. (https://github.com/opensearch-project/OpenSearch/blob/737843fcc447c4e9167edc1e24a7ed59c043241c/server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionTests.java#L537)

Reply:
Are we planning to rename the file name as well?

Reply:
The `MasterRole` here is used intentionally, but I will rename the `Master` word in `testDelegateToMasterOnNode()` 😁 Good catch.

Reply:
Updated in the commit https://github.com/opensearch-project/OpenSearch/pull/2520/commits/a87cede9b88976e0a043e7ea8670d3de2f400c1e